apps/examples/webserver_test/webserver_test_util.c

```diff
@@ -303,39 +307,24 @@ void parse_body(struct http_message_len_t *len, char *buf, int buf_len,
 					 struct http_client_response_t *response, int *state,
 					 char **body, int *read_finish, int *process_finish)
 {
-	bool is_header = false;
 	if (!len->message_len) {
-		is_header = true;
 		*body = buf + len->sentence_start;
 	}
-	if (buf_len - len->sentence_start + len->message_len == len->content_len) {
+
+	len->message_len = buf_len;
+
+	if (buf_len - len->sentence_start == len->content_len) {
 		buf[buf_len] = '\0';
 		if (response) {
 			response->total_len = len->content_len;
 
-			if (is_header == true) {
 				response->entity_len = buf_len - len->sentence_start;
 				response->entity = buf + len->sentence_start;
-			} else {
-				response->entity_len = buf_len;
-				response->entity = buf;
-			}
+				printf("response entity: %s\n",response->entity);
```

Comment:
Please use PRNT instead of printf. 
Also, please provide space after comma at all places. 
Use capitals for the starting of the sentence.

Reply:
done